src/ns/adapters/outbound/lox24_client.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2021 - 2025 Universität Tübingen, DKFZ, EMBL, and Universität zu Köln
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Contains the sms client adapter"""
+
+import logging
+
+from httpx import HTTPStatusError, Response, post
+from pydantic import Field, PositiveFloat, SecretStr
+from pydantic_settings import BaseSettings
+
+from ns.ports.outbound.sms_client import SmsClientPort
+
+log = logging.getLogger(__name__)
+
+
+class Lox24ClientConfig(BaseSettings):
+    """Configuration details for the Lox24Client"""
+
+    lox24_host: str = Field(
+        default="api.lox24.eu", description="The address of the lox24 API"
+    )
+    lox24_port: int = Field(default=443, description="The port of the lox24 API")
+    lox24_token: SecretStr = Field(default=..., description="The authentication token")
+
+    lox24_timeout: PositiveFloat | None = Field(
+        default=10,
+        description=(
+            "The maximum amount of time (in seconds) to wait for a connection to the"
+            + " lox24 API. If set to `None`, the operation will wait indefinitely."
+        ),
+    )
+    lox24_sms_send_path: str = Field(
+        default="/sms", description="The path for sending SMS messages"
+    )
```

Comment:
```suggestion
    lox24_send_url: HttpUrl = Field(
        default="https://api.lox24.eu:443/sms",
        description="The URL to use for sending SMS messages via Lox24",
    )
    lox24_token: SecretStr = Field(default=..., description="The authentication token")

    lox24_timeout: PositiveFloat | None = Field(
        default=10,
        description=(
            "The maximum amount of time (in seconds) to wait for a connection to the"
            + " lox24 API. If set to `None`, the operation will wait indefinitely."
        ),
    )
```
See comment below about consolidating these parameters. The string format can be automatically validated by the `HttpUrl` type provided by pydantic